src/Registry/Registry.jl

```diff
@@ -414,7 +414,7 @@ function update(regs::Vector{RegistrySpec} = RegistrySpec[]; io::IO=stderr_f(),
                         GitTools.fetch(io, repo; refspecs=["+refs/heads/$branch:refs/remotes/origin/$branch"])
                     catch e
                         e isa Pkg.Types.PkgError || rethrow()
-                        push!(errors, (reg.path, "failed to fetch from repo"))
+                        push!(errors, (reg.path, "failed to fetch from repo. Error: $(e.msg)"))
```

Comment:
```suggestion
                        push!(errors, (reg.path, "failed to fetch from repo: $(e.msg)"))
```
maybe? I think that is what is used elsewhere.